tox.ini

```diff
@@ -53,14 +53,32 @@ commands =
   # Saves a report to coverage.xml (uploaded to Codecov from CI, as Codecov apparently does not accept .coverage binary files)
   coverage xml
 
-[testenv:docs]
-description = "Build documentation"
+[testenv:build_sphinx_docs]
+description = "Build sphinx documentation"
 usedevelop=true
 deps =
-  -r{toxinidir}/requirements.docs.txt
+  -r{toxinidir}/requirements.sphinx_docs.txt
 commands =
   sphinx-build -W --keep-going -b html docs/source docs/html
 
+[testenv:build_cmd_docs]
+description = "Build command line documentation"
+usedevelop=true
+deps =
+  -r{toxinidir}/requirements.cmd_docs.txt
+commands =
+  python docs/generate_docs.py
+
+
+[testenv:check_docs]
+description = "Check if command line documentation is up to date with sphinx documentation"
+usedevelop=true
```

Comment:
I don't think `true` (i.e. installing the package into the venv) is needed here... there's an entire task to smooth that out, #323

Reply:
@PawelLipski Checking if documentation is up-to-date uses docs generating script which in turn uses git_machete package so it has to stay.